src/app/pages/system/network/components/default-gateway-dialog/default-gateway-dialog.component.html

```diff
@@ -18,6 +39,18 @@ <h1 mat-dialog-title>{{ 'Register Default Gateway' | translate }}</h1>
         [required]="true"
         [tooltip]="helptext.ipv4gatewayTooltip | translate"
       ></ix-input>
+      
+      <ix-input
+        formControlName="dns1"
+        [label]="'DNS Server #1' | translate"
```

Comment:
The label can be improved by just stating "Primary DNS Server" or maybe mentioning "Primary" in the label somewhere e.g., "DNS Server #1 (Primary)" to avoid a click to see the tooltip to know that's what it is. Same for the secondary DNS input below.